src/btreemap.rs

```diff
@@ -1494,6 +1552,26 @@ mod test {
         });
     }
 
+    #[test]
+    fn pop_last_simple() {
```

Comment:
Can we also add separate tests where we call `pop_last` and `pop_first` on an empty btree?